_posts/2019-01-XX-meltdown-and-spectre-vulnerabilities.md

```diff
@@ -0,0 +1,150 @@
+---
+post_title: Meltdown spectre vulnerabilities
```

Comment:
@miczyg1 let's improve this title. "What you should know about Spectre and Meltdown before patching?" - 20%

Reply:
Volume: 22,200/mo | CPC: $4.09 | Competition: 0.04
for `meltdown and spectre`. As these are the first words appearing in the title, it should be good to go now. Changed title to `Meltdown and Spectre. What are they and what are they not?`

Reply:
@miczyg1 super.